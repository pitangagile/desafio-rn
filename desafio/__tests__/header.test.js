import React from 'react';
import HeaderApp from '../src/components/header';

import renderer from 'react-test-renderer';

test('Header Snapshot', () => {
  const tree = renderer
    .create(<HeaderApp title="Aqui vai um titulo" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
