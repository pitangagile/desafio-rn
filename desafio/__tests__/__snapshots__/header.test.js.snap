// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Snapshot 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#2f425a",
        "height": 50,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#fff",
          "fontSize": 20,
          "fontWeight": "bold",
        }
      }
    >
      Aqui vai um titulo
    </Text>
  </View>
</RCTSafeAreaView>
`;
